src/esp/bindings/SensorBindings.cpp

```diff
@@ -272,7 +274,102 @@ void initSensorBindings(py::module& m) {
              Magnum::SceneGraph::PyFeatureHolder<FisheyeSensor>>(
       m, "FisheyeSensor")
       .def(py::init_alias<std::reference_wrapper<scene::SceneNode>,
-                          const FisheyeSensorSpec::ptr&>());
+                          const FisheyeSensorSpec::ptr&>())
+      .def(
+          "buffer",
+          [](CameraSensor& self, int gpuDevice) {
+            py::handle handle = py::cast(self);
```

Comment:
```suggestion
          [](py::handle handle, int gpuDevice) {
            auto self = handle.cast<CameraSensor>();
```
Keeps the original Python object around instead of casting impliciting it a C++ object back again. This instead casts the raw python handle to a C++ object once allow you to reuse the original handle. 
Although I would consider renaming these variables to be a bit less confusing. IE. self_handle instead of just handle.
See #1858 for an example on how to do it.

Reply:
Good catch.